third_party/musl/src/string/memccpy.c

```diff
@@ -7,6 +7,9 @@
 #define HIGHS (ONES * (UCHAR_MAX/2+1))
 #define HASZERO(x) ((x)-ONES & ~(x) & HIGHS)
 
+#ifdef __GNUC__
```

Comment:
Is it worth carving this out into a shared macro somewhere ? like `#MAYBE_ATTRIB_NO_SANITIZE_ADDRESS` ?